ILTransform/Program.cs

```diff
@@ -158,6 +158,10 @@ public static int Main(string[] args)
                 {
                     testStore.FixILFileNames();
                 }
+                else if(settings.DeduplicateProjectNames)
```

Comment:
```suggestion
                else if (settings.DeduplicateProjectNames)
```

I'm curious - what editor/IDE do you use?  If I use VS (with I think default configuration), then I automatically get a style enforced for consistency (which then makes these things jump out at me, sigh..)

Reply:
I am using VS 2022 but I am getting none of these recommendations or autoformat, maybe I have them disabled, I will check it